### PR TITLE
Add info about OpenTel collector option 

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?
Adds info about using the Datadog exporter for OpenTel Collector.

### Motivation
DOCS-1409

### Preview

https://docs-staging.datadoghq.com/kari/open-tel-intro/tracing/setup_overview/open_standards/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/setup_overview/open_standards/_index.md [hunk 1]
```diff
@@ -2,9 +2,98 @@
...
+    traces:
+      receivers: [examplereceiver]
+      processors: [batch]
+      exporters: [datadog/api]
+```
+
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
```
Comment (ericmustin) [inline content/en/tracing/setup_overview/open_standards/_index.md:94]: I realise now i inlined a lot of markdown links that belong as [] thingys instead. sorry about that.

File: content/en/tracing/setup_overview/open_standards/_index.md [hunk 2]
```diff
@@ -2,9 +2,98 @@
...
+<!--- 1. Hostname set by another OpenTelemetry component -->
+1. Manually set hostname in configuration
+1. EC2 non-default hostname (if in EC2 instance)
+1. EC2 instance id (if in EC2 instance)
+1. Fully qualified domain name
+1. Operating system host name
+
+### Ingesting OpenTelemetry Traces with the Collector
+
+The exporter assumes you have a pipeline that not only uses the exporter, but also includes a batch processor configured with the following:
```
Comment (ericmustin) [inline content/en/tracing/setup_overview/open_standards/_index.md:52]:
```suggestion
The OpenTelemetry Collector is configured by adding a [pipeline](https://github.com/open-telemetry/opentelemetry-collector/blob/master/docs/design.md#pipelines) to your `otel-collector-configuration.yml` file. The relative path to this configuration file should be supplied when starting the collector by passing it in via the `--config=<path/to/configuration_file>` command line argument. Examples of supplying a configuration file can be found in the [Opentelemetry Collector documentation](https://github.com/open-telemetry/opentelemetry-collector/tree/master/examples).  The exporter assumes you have a pipeline that not only uses the datadog exporter, but also includes a [batch processor](https://github.com/open-telemetry/opentelemetry-collector/tree/master/processor/batchprocessor#batch-processor) configured with the following:
```

**Review by ericmustin — CHANGES_REQUESTED** [review]
Hey @kayayarai , thanks for the great work here. This is really thorough and an excellent distillation of a number of vague and disparate docs. I think overall this is strong, if push comes to shove it's probably fine to ship as is. I tried to add my feedback and suggestions where I felt appropriate, and add in technical detail and clarification where i could. but i certainly defer to your opinion here. Please feel free to let me know if any feedback is unclear and ignore anything you don't think is accurate or relevant, i tried to be overly verbose here so that you would have enough context to make intelligent choices about what to add/keep/ignore.

File: content/en/tracing/setup_overview/open_standards/_index.md (hunk 1 above)
Comment (kayayarai) [reply content/en/tracing/setup_overview/open_standards/_index.md:101]: lol, nah, all good. I appreciate getting the good info and can take care of the formatting.

File: content/en/tracing/setup_overview/open_standards/_index.md [hunk 3]
```diff
@@ -2,9 +2,98 @@
...
+      site: datadoghq.eu
+
+service:
+  pipelines:
+    traces:
+      receivers: [examplereceiver]
+      processors: [batch]
+      exporters: [datadog/api]
+```
+
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/tracing/setup_overview/open_standards/_index.md:95]: Good idea. Added.

File: content/en/tracing/setup_overview/open_standards/_index.md [hunk 4]
```diff
@@ -2,9 +2,98 @@
...
+1. Fully qualified domain name
+1. Operating system host name
+
+### Ingesting OpenTelemetry Traces with the Collector
+
+The exporter assumes you have a pipeline that not only uses the exporter, but also includes a batch processor configured with the following:
+  - A `timeout` setting of `10s` (10 seconds). A batch representing 10 seconds of traces is a constraint of Datadog's API Intake for Trace Related Statistics. Without this setting, trace related metrics including `.hits`, `.errors`, and `.duration` for different services and service resources may be inaccurate over periods of time.
+
+<div class="alert alert-warning">
+The Datadog exporter for the OpenTelemetry Collector is currently in beta. It may consume high CPU and memory resources. Configuring particularly the pipeline and batch processor may take some iteration before it responds with accurate metrics given your production environment. <a href="https://docs.datadoghq.com/help/">Reach out to support</a> if it doesn't work as you expect.
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/tracing/setup_overview/open_standards/_index.md:61]: 🙇‍♀️

File: content/en/tracing/setup_overview/open_standards/_index.md [hunk 5]
```diff
@@ -2,9 +2,98 @@
...
+1. Manually set hostname in configuration
+1. EC2 non-default hostname (if in EC2 instance)
+1. EC2 instance id (if in EC2 instance)
+1. Fully qualified domain name
+1. Operating system host name
+
+### Ingesting OpenTelemetry Traces with the Collector
+
+The exporter assumes you have a pipeline that not only uses the exporter, but also includes a batch processor configured with the following:
+  - A `timeout` setting of `10s` (10 seconds). A batch representing 10 seconds of traces is a constraint of Datadog's API Intake for Trace Related Statistics. Without this setting, trace related metrics including `.hits`, `.errors`, and `.duration` for different services and service resources may be inaccurate over periods of time.
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/tracing/setup_overview/open_standards/_index.md]: Done.

File: content/en/tracing/setup_overview/open_standards/_index.md (hunk 2 above)
Comment (kayayarai) [reply content/en/tracing/setup_overview/open_standards/_index.md]: Much better, thank you!

File: content/en/tracing/setup_overview/open_standards/_index.md [hunk 6]
```diff
@@ -2,9 +2,98 @@
...
+```
+To send the data to the Datadog EU site, also set the `site` parameter:
+```
+datadog:
+  api:
+    key: "<API key>"
+    site: datadoghq.eu
+```
+
+On each OpenTelemetry-instrumented application, set the resource attributes `development.environment`, `service.name`, and `service.version` using the language's SDK. As a fall-back, you can also configure hostname (optionally), environment, service name, and service version at the collector level for unified service tagging by following the [example configuration file][5]. If you don't specify the hostname explicitly, the exporter attempts to get an automatic default by checking the following sources in order, falling back to the next one if the current one is unavailable or invalid:
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/tracing/setup_overview/open_standards/_index.md]: Agreed, added the link for now.

File: content/en/tracing/setup_overview/open_standards/_index.md [hunk 7]
```diff
@@ -2,9 +2,98 @@
...
+
+Click your language to see instructions for setting these up:
 
 {{< partial name="apm/apm-opentracing.html" >}}
 
+<br>
+
+## OpenTelemetry Collector
+
+Alternatively, Datadog has [an exporter that uses the OpenTelemetry Collector][3] to receive traces and metrics data from the OpenTelemetry SDKs, and to forward the data on to Datadog (without the Agent). It works with all supported languages, and you configure it by supplying the [Datadog API key][4]:
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/tracing/setup_overview/open_standards/_index.md]: Hardly a minor nit! Needed the help here. Thank you. I've slightly reworded, but this is much better

File: content/en/tracing/setup_overview/open_standards/_index.md [hunk 8]
```diff
@@ -2,9 +2,98 @@
...
+  text: "Collector documentation"
+- link: "https://www.datadoghq.com/blog/opentelemetry-instrumentation/"
+  tag: "Blog"
+  text: "Datadog's partnership with OpenTelemetry"
 aliases:
+
+
 ---
-Datadog supports a variety of open standards, including OpenTracing and OpenTelemetry. For language-specific instructions, select your language:
+Datadog supports a variety of open standards, including [OpenTracing][1] and [OpenTelemetry][2]. Each of the following languages has support for sending Open Tracing data to Datadog. Python, Ruby, and Go also have OpenTelemetry Datadog span exporters, which export traces from OpenTelemetry to the Datadog Agent.
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/tracing/setup_overview/open_standards/_index.md]: yeah, I agree this was/is confusing, and I think eventually we might be better off moving the Python, Ruby, NodeJS OTel span exporters info out from beside the Open Tracing info. That would make this intro clearer at least. For now, I did some rewording as you suggest

**Review by ericmustin — APPROVED** [review]
👍Nice work! LGTM from the APM+OpenTelemetry side of things.

**Review by apigirl — APPROVED** [review]

**Pull request merged**